File: VB/LoadDataFromDocx/MauiProgram.vb
Imports Microsoft.Extensions.Logging
Imports DevExpress.Maui

Namespace HtmlEditLoadDataFromDocx

    Public Module MauiProgram

        Public Function CreateMauiApp() As MauiApp
            Dim builder = MauiApp.CreateBuilder()
            builder.UseMauiApp(Of App)().UseDevExpress().ConfigureFonts(Function(fonts)
                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular")
                fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold")
            End Function)
#If DEBUG
            builder.Logging.AddDebug()
#End If
            Return builder.Build()
        End Function
    End Module
End Namespace

File: CS/HtmlToMarkdown/MauiProgram.cs
using DevExpress.Maui;
using Microsoft.Extensions.Logging;
using Microsoft.Maui.Controls.Hosting;
using Microsoft.Maui.Hosting;
using DotNet.Meteor.HotReload.Plugin;

namespace HTMLtoMD;

public static class MauiProgram
{
	public static MauiApp CreateMauiApp()
	{
		var builder = MauiApp.CreateBuilder();
		builder
			.UseMauiApp<App>()
			.UseDevExpress()
            .UseDevExpressControls()
            .UseDevExpressCollectionView()
            .UseDevExpressEditors()
            .UseDevExpressHtmlEditor()
#if DEBUG
            .EnableHotReload() //DEMO_REMOVE
#endif
            .ConfigureFonts(fonts =>
			{
				fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
				fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
			});

#if DEBUG
		builder.Logging.AddDebug();
#endif

		return builder.Build();
	}
}
